src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
```

Comment:
I'm ok with adjusting this, but as said in that other PR would really like to have an `eslint` rule for this. We're wasting way to much time with manually enforcing style! Either there is a rule or it shouldn't be considered an error – let's focus on the important parts and shipping features.